tests/register_operation_tests.rs

```diff
@@ -58,7 +58,7 @@ proptest!(
         state7 in any::<bool>(),
         state8 in any::<bool>()
     ) {
-        let mut register = Register::new([state, state2, state3, state4, state5, state6, state7, state8]);
+        let mut register = Register::new(vec![state, state2, state3, state4, state5, state6, state7, state8]);
```

Comment:
We should refactor these tests later as they can be dynamically sized now.
This does not need to be handled by this issue though.